### PR TITLE
[WIP] - Draft PR - pneumoniamnist datasetP

### PR DESCRIPTION
**Pull request by CristianoPizzamiglio**
**Draft PR**

The code is not complete, I would like to get feedback, please. If you are interested I will continue the development.

The lightweight Pneumonia MNIST dataset could be useful for beginners in supervised learning interested in training neural networks for biomedical applications.

# Add Dataset

* Dataset Name: Pneumomia mnist
* Info: https://medmnist.com/
* Data: https://zenodo.org/records/10519652

## Description

The PneumoniaMNIST is based on a prior dataset of 5,856 pediatric chest X-Ray images. The task is binary-class classification of pneumonia against normal. The source training set is split with a ratio of 9:1 into training and validation set, and use its source validation set as the test set. The source images are gray-scale, and their sizes are (384–2,916) × (127–2,713). The images are center-cropped with a window size of length of the short edge and resized into 1 × 28 × 28.

### CODE REVIEW

**Comment by ccl-core** [comment]
hi Cristiano, thanks for your contribution!

This looks good :) Can you just:

1) Move the dataset from the `image_classification` to the `dataset` folder
2) Remove all non-relevant tags from TAGS.txt
3) Add fake data to dummy_data

As an example of a PR adding a dataset to the correct folder and with the correct files, you can have a look at: https://github.com/tensorflow/datasets/commit/5227bedafd7ae27da5733c3bc714d2b2fab437f3

**Comment by ccl-core** [comment]
Also, feel free to mark the PR as `Ready for review` :)

**Comment by CristianoPizzamiglio** [comment]
Thanks! Soon the PR will be ready for review.

**Comment by ccl-core** [comment]
Thank you, @CristianoPizzamiglio . You PR will be merged soon!
